ogre2/src/media/materials/programs/skybox_fs.glsl

```diff
@@ -0,0 +1,16 @@
+#version 330
```

Comment:
I wonder if we should be putting copyright headers in these files.

Reply:
these were adapted from ogre samples so added their license header and a note 0823885